src/status_im2/contexts/wallet/wallet/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im2.contexts.wallet.wallet.view
```

Comment:
Should we rename this ns to `status-im2.contexts.wallet.home.view`? to have consistency with other tabs?
```suggestion
(ns status-im2.contexts.wallet.home.view
```

Reply:
Yep, sounds good @smohamedjavid !